tilelang/_typing.py

```
@@ -27,6 +27,8 @@
 # In Python 3.9, we can only use isinstance(a, (TypeA, TypeB, ...)) instead of isinstance(a, TypeA | TypeB | ...))
 BufferLikeTypeTuple = (tir.Buffer, BufferLoad, BufferRegion)
 
+CompileUnitResult = tuple[int, dict[str, Any], Optional[JITKernel], Optional[Exception]]
```

Ruff (0.15.12)

[error] Undefined name `Any`

(F821)